Makefile

```diff
@@ -85,6 +95,9 @@ windows-x64.test: windows-x64 test/run.py
 	$(DOCKER) run --rm dockcross/windows-x64 > $(BIN)/dockcross-windows-x64 && chmod +x $(BIN)/dockcross-windows-x64
 	$(BIN)/dockcross-windows-x64 python test/run.py --exe-suffix ".exe"
 
+Dockerfile: Dockerfile.in common.docker
+	cpp -o Dockerfile Dockerfile.in
```

Comment:
This is to prevent duplication of *common.docker*. I am not sure how this could be done with sed.